runtime.md

```diff
@@ -30,6 +30,15 @@ When serialized in JSON, the format MUST adhere to the following pattern:
 }
 ```
 
+```json
+{
+    "ociVersion": "0.2.0",
+    "id": "oci-container1",
+    "pid": 4422,
```

Comment:
Probably worth waiting until #459 settles for this.

And it would be good to get [anchor-headers][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/4a1a84096cbc4f21af6a92cb3ef1ff3c3f1e6d45/style.md#examples